run_runner_locally.sh

```diff
@@ -6,7 +6,10 @@ set -e
 
 # Settings you might want to change
 export PYTHON_BINARY=python3.9
-export PORT=5000
+export PORT=6000  # if you change this, update port_mapping in mirrord.json too
```

Comment:
Was there a reason for changing the port?

Reply:
That port is already taken on my laptop, I think it happened to someone else too. If you don't mind, I'd like to jjust change the default.